discovery-provider/src/tasks/index_spl_token.py

```diff
@@ -170,53 +174,87 @@ def parse_spl_token_transaction(
                 f"index_spl_token.py | {tx_sig} No transfer checked instruction found"
             )
             return None
+
         tx_message = cast(Transaction, result.transaction.transaction).message
-        instruction = get_valid_instruction(tx_message, meta, SPL_TOKEN_ID)
-        if not instruction:
-            logger.error(
-                f"index_spl_token.py | {tx_sig} No valid instruction for spl token program found"
-            )
-            return None
+        account_keys = list(map(lambda key: str(key), tx_message.account_keys))
+        if version == 0 and meta.loaded_addresses:
+            writable = meta.loaded_addresses.writable or []
+            readonly = meta.loaded_addresses.readonly or []
+            account_keys.extend(str(key) for key in writable + readonly)
```

Comment:
I'm 98% sure this is correct, but I couldn't find proper documentation.
https://docs.solana.com/integrations/exchange#versioned-transaction-migration

Things may be in loaded addresses, not account keys ... but like ... is it writable + readonly or readonly + writable?

Reply:
writeable + readonly it looks like

https://github.com/solana-labs/solana-web3.js/blob/958cc6b5ceaa8eddc79d46ed133e2ceac4593476/packages/library-legacy/src/message/v0.ts#L385C1-L403C11

Reply:
🙇